input/resources/ValueSet-ehealth-sor-organization-type.json

```diff
@@ -1,21 +1,25 @@
 {
     "resourceType": "ValueSet",
     "id": "ehealth-sor-organization-type",
+    "text": {
```

Comment:
Narrative (text element) must not be included. It is generated when imported into terminology